webapp/cube/views.py

```diff
@@ -275,6 +275,53 @@ def get_microcerts():
     )
 
 
+@login_required
+def post_microcerts_purchase():
+    """
+    Purchase preview for CUBE microcertifications
+    """
+    preview = flask.request.args.get("preview", False)
+    account_id = flask.request.json["account_id"]
+    # Only purchase of one item allowed at a time
+    product_listing_id = flask.request.json["product_listing_id"]
+    previous_purchase_id = flask.request.json["previous_purchase_id"]
+    value = flask.request.json["value"]
+
+    is_test_backend = (
```

Comment:
This is a really confusing block. So you set the `is_test_backend` variable, which I would expect to contain a boolean, to the value of either the `CONTRACTS_TEST_API_URL` or the `CONTRACTS_LIVE_API_URL`, which I'm pretty sure is a string?

I imagine you've copied this pattern from somewhere else, but at least for this function could you please rename this variable to something that makes sense to contain a URL?

Reply:
Yeah, I am basically using the same system as advantage to make it consistent, but removed the complexity of using the global store `flask.g.api` which I find it even worse (because we can use the current context in this case, no need for the global context). 
If we want to change the way we handle these 2 URLs, I think we should do it in both advantage and here too.

Reply:
Changed it to your liking :). But yeah I agree about the variable name, didn't really notice.